python/tirex_tracker/__init__.py

```diff
@@ -966,15 +1039,15 @@ def tracking(
     poll_intervall_ms: int = -1,
     system_name: Optional[str] = None,
     system_description: Optional[str] = None,
-    export_file_path: Optional[PathLike] = None,
+    export_file_path: Optional[PathLike] | str = None,
```

Comment:
The ` | ` type union operator won't work on all Python versions. But it should also not be necessary to change the type annotation here, as `PathLike` should already allow `str`.